From 3edad11bd21c34cbca50ab11b7240333b1f2fb1b Mon Sep 17 00:00:00 2001
From: Alexandr Nedvedicky <sashan@openssl.org>
Date: Fri, 1 Mar 2024 08:25:19 +0100
Subject: [PATCH 027/228] FAQ.md should be removed

the page the link refers to does not exist.
Anyone objects to delete file?

Reviewed-by: Tomas Mraz <tomas@openssl.org>
Reviewed-by: Matt Caswell <matt@openssl.org>
(Merged from https://github.com/openssl/openssl/pull/23719)

(cherry picked from commit 854539889d31ed2ea63280256fd7aab66e828ae5)
---
 FAQ.md | 6 ------
 1 file changed, 6 deletions(-)
 delete mode 100644 FAQ.md

diff --git a/FAQ.md b/FAQ.md
deleted file mode 100644
index 30f5010ce3..0000000000
--- a/FAQ.md
+++ /dev/null
@@ -1,6 +0,0 @@
-Frequently Asked Questions (FAQ)
-================================
-
-The [Frequently Asked Questions][FAQ] are now maintained on the OpenSSL homepage.
-
-  [FAQ]: https://www.openssl.org/docs/faq.html
-- 
2.39.5 (Apple Git-154)

